blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/PipelineFolderImpl.java

```diff
@@ -97,7 +97,7 @@ public BlueFavorite favorite(@JsonBody BlueFavoriteAction favoriteAction) {
 
     @Override
     public Link getLink() {
-        return OrganizationImpl.INSTANCE.getLink().rel("pipelines").rel(PipelineImpl.getRecursivePathFromFullName(this));
+        return OrganizationImpl.INSTANCE.getLink().rel("pipelines").rel(AbstractPipelineImpl.getRecursivePathFromFullName(this));
```

Comment:
I am sure I have asked before, but how can something be both abstract and a concrete impl at the same time?